docs/release-checklist.md

```diff
@@ -91,7 +104,13 @@ on https://developer.amazon.com/dashboard :
     d) hit "Submit app" at the upper right corner
 
 
-# Testing checklist
+## Releases on Apklis, Passkoocheh
+
+These stores are not under our control.
```

Comment:
If you want I can create the account and then ask you for the info and tell you the command you must run to generate the token/hash they ask for, we can coordinate this in DC, then with that account created it can be used to upload DC directly without 3rd parties

Reply:
ah, okay, did not know that. maybe to go for it. let's discuss that if there is a bit more room for it, these days are already too many things ongoing.

however, from the current status, this is still true, i (or whoever do releases) have _currently_ no access :)